File: 백준/Bronze/2525. 오븐 시계/오븐 시계.py
h, m = map(int, input().split())
time = int(input())

resm = m + time
resh = h
while resm >= 60:
    resh = resh + 1
    resm = resm - 60
    
    if resh >= 24:
        resh = resh - 24


print(resh,resm)

File: 백준/Bronze/2525. 오븐 시계/오븐 시계.cc
#include <stdio.h>

int main(void){
    int nowH, nowM, time, resH, resM;

    nowH = 0;
    nowM = 0;
    time = 0;

    scanf("%d %d", &nowH, &nowM);
    scanf("%d", &time);

    resH = nowH;
    resM = nowM + time;

    while(resM >= 60){
        resH += 1;
        resM -= 60;

        if(resH >= 24){
            resH -= 24;
        }
    }

    printf("%d %d\n", resH, resM);
    return 0;
}